activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/decorators/ProjectDecorator.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
The code related to decorators and filters adds the possibility to implement those capabilities for getProject and getProjects endpoints.